README.md

```diff
@@ -661,6 +661,31 @@ Example:
     db/migrate/1234_create_my_table.rb
     app/models/my_table.rb
 
+#### Generating Tables, Models and Consumer classes from schema
+
+Deimos provides a generator that takes an existing schema and generates a
+database table based on its fields, updates the specified kafka configuration file,
+and creates a consumer class.
+
+Before running this migration, you must first copy the schema into your repo
+in the correct path (in the example above, you would need to have a file
+`{SCHEMA_ROOT}/com/my-namespace/MySchema.avsc`).
+
+To generate a model and migration, run the following:
+
+    rails g deimos:kafka_consumer TABLE_NAME FULL_SCHEMA_NAME TOPIC_NAME KAFKA_CONFIG_FILE_PATH
```

Comment:
For that many arguments I don't think positional arguments is going to be usable. We should use optional arguments, and provide sane defaults so people don't have to specify them all the time. Preferably something like:

`rails g deimos:kafka_consumer TOPIC_NAME FULL_SCHEMA_NAME`

where we could infer the table name by just taking the name of the schema (e.g. if the schema type is called "FlyerItem" we could create a table "flyer_items"). I think we could even figure out which initializer file does the configuration by searching the files in `config/initializers` for `Deimos.configure`. (We can override this in FlippRubyKafka to work for our version as well.)